### ISSUE TITLE
Version Update for UnrealEngine 5.4.4

### ISSUE THREAD

**Issue by DocMasterRMT** [oculus-samples/Unreal-HandPoseShowcase#5]
the actual version of the Plugin (v69) is not compatible with latest version of the engine (5.4.4)

![image](https://github.com/user-attachments/assets/270524b6-f27f-4151-bdd9-8fa4fd568ffc)

**Comment by animandivr** [comment]
any plan to update to 5.4.4 ?

**Comment by G2GVR** [comment]
Hello! Any chance for the 5.4.4 ? thank you

**Comment by JuanseLandeta** [comment]
No response for the update to 5.4.4?

**Comment by goingafk** [comment]
I think this plugin is now being replaced with the MetaInteractionSDK. The lack of documentation doesn't help but it seems like this is the way that interaction is going and I found it has been better for more simpler interactions

**Comment by KutlayTunc** [comment]
I'm on 5.5.3 and I can compile this project. But if I take plugin out of this project to another I get compile errors.

**Comment by praful0203** [comment]
Any update on this? Are we able to compile on at least 5.4.4?
Would be grateful if it could run on 5.5!

**Comment by KutlayTunc** [comment]
@praful0203 https://github.com/oculus-samples/Unreal-HandPoseShowcase/issues/6#issue-2855486436 I’ve managed it to be working with this solution, I hope it helps

**Comment by praful0203** [comment]
> [@praful0203](https://github.com/praful0203) [#6 (comment)](https://github.com/oculus-samples/Unreal-HandPoseShowcase/issues/6#issue-2855486436) I’ve managed it to be working with this solution, I hope it helps

FYI, this change does not fix the error, even in UE 5.3.4 source build.
I think for now, we just have to comment the UE_Log statement unless this gets fixed properly.

![Image](https://github.com/user-attachments/assets/0e3b227d-c4c3-47f2-ace8-d0063b925e29)

**Comment by KutlayTunc** [comment]
Yeah because your error is different than mine, I hope you find your solution.

**Comment by AbdElhalim12** [comment]
> > [@praful0203](https://github.com/praful0203) [#6 (comment)](https://github.com/oculus-samples/Unreal-HandPoseShowcase/issues/6#issue-2855486436) I’ve managed it to be working with this solution, I hope it helps
> 
> FYI, this change does not fix the error, even in UE 5.3.4 source build. I think for now, we just have to comment the UE_Log statement unless this gets fixed properly.
> 
> ![Image](https://github.com/user-attachments/assets/0e3b227d-c4c3-47f2-ace8-d0063b925e29)

Try this:

```
for (const FHandGestureStep& TimedPose : TimedPoses)
{
    TCHAR IndicatorChar = (CurrentStep == Step) ? TCHAR('>') : TCHAR(' ');

    UE_LOG(LogHandPoseRecognition, Display, TEXT(" %c %d %s[%d] %05.3f - %05.3f"),
        IndicatorChar,
        Step,
        *HandPoseRecognizer->Poses[TimedPose.PoseIndex].PoseName,
        TimedPose.PoseIndex,
        TimedPose.StepFirstTime,
        TimedPose.StepLastTime);

    ++Step;
}
```
